test-fixture.html

```diff
@@ -313,9 +315,20 @@
     });
 
   try {
-    document.registerElement('test-fixture', {
-      prototype: TestFixturePrototype
-    });
+    if (window.customElements) {
+      function TestFixture() {
+        if (window.Reflect) {
+          return Reflect.construct(HTMLElement, [], TestFixture)
+        }
+        return HTMLElement.call(this) || this;
```

Comment:
You might need to wrap this in a try-catch rather than `||` since `HTMLElement.call(..)` throws with native `HTMLElement`.